skimage/transform/_geometric.py

```diff
@@ -164,22 +164,18 @@ def __call__(self, coords):
         """
         raise NotImplementedError()
 
-    def inverse(self, coords):
-        """Apply inverse transformation.
-
-        Parameters
-        ----------
-        coords : (N, 2) array
-            Destination coordinates.
-
-        Returns
-        -------
-        coords : (N, 2) array
-            Source coordinates.
-
-        """
+    @property
+    def inverse_params(self):
         raise NotImplementedError()
 
+    @property
+    def inverse(self):
+        """Return the inverse transformation."""
+        # ASK: Can you make a property a class method too?
```

Comment:
If you're referring to [_warps.py:830](https://github.com/scikit-image/scikit-image/pull/3077/files?utf8=%E2%9C%93&diff=unified#diff-c0f33f5600401be736e79b65164b4c81R839), then as explained, this shouldn't be an issue.